scripts/demo.ts

```diff
@@ -62,7 +62,23 @@ dbBot.slots = slots;
 // load csv file //
 ///////////////////
 
-dbBot.loadFile("./pokemon.csv");
+dbBot.loadFile("./sw_characters.csv");
+
+/////////////////////////
+// null values in rows //
+/////////////////////////
+
+const nullValues = [null, "NA", NaN];
```

Comment:
In the future this should be a solid demo, and will not be changed for every tryout